website/docs/r/compute_http_health_check.html.markdown

```diff
@@ -10,11 +10,13 @@ description: |-
 
 Manages an HTTP health check within GCE. This is used to monitor instances
 behind load balancers. Timeouts or HTTP errors cause the instance to be
-removed from the pool. For more information, see [the official
-documentation](https://cloud.google.com/compute/docs/load-balancing/health-checks)
+removed from the pool. Creates an HttpHealthChecks resource. For more information, see [the official
```

Comment:
This addition seems confusing to me, and doesn't really clarify anything. I think we could do without it.

Reply:
Fixed.

Reply:
Think you may have forgotten to push a commit?

Reply:
Sorry, pushed now.